app/src/js/components/Header/header.js

```diff
@@ -33,8 +33,10 @@ class Header extends React.Component {
 
   componentDidMount () {
     const { dispatch, api } = this.props;
-    if (api.authenticated) dispatch(getApiVersion());
-    dispatch(getCumulusInstanceMetadata());
+    if (api.authenticated) {
+      dispatch(getApiVersion());
+      dispatch(getCumulusInstanceMetadata());
```

Comment:
This is unrelated to the rest of the changes. However, I kept getting an error message of "Authorization header missing" on my homepage when initially loading the dashboard. 

I tracked it down to this header component making the request for instance metadata, even when the user is not logged in. Since the /instanceMeta endpoint does require authentication, this change seemed reasonable and made the error go away for me. 

I can exclude this change if there's some reason it shouldn't be made though.